addons/account/wizard/account_move_send.py

```diff
@@ -326,25 +336,8 @@ def _format_error_html(self, error):
     # -------------------------------------------------------------------------
 
     def action_open_partners_without_email(self, res_ids=None):
-        partners = self.move_ids.mapped("partner_id").filtered(lambda x: not x.email)
-        if len(partners) == 1:
-            return {
-                'type': 'ir.actions.act_window',
-                'res_model': 'res.partner',
-                'view_mode': 'form',
-                'target': 'current',
-                'res_id': partners.id,
-            }
-        else:
-            return {
-                'type': 'ir.actions.act_window',
-                'res_model': 'res.partner',
-                'view_mode': 'tree,form',
-                'target': 'current',
-                'name': _('Partners without email'),
-                'context': {'create': False, 'delete': False},
-                'domain': [('id', 'in', partners.ids)],
-            }
+        # TODO: remove this method in master
+        return self.env['ir.ui.view'].get_records_action(self, name=_("Partners without email"))
```

Comment:
🤔 
self is account.move.send records, not filtered partners

Reply:
Another bad case of Friday afternoon.